recipe/conda_build_config.yaml

```diff
@@ -422,6 +422,8 @@ vlfeat:
   - 0.9.20
 vtk:
   - 8.1.1
+x264:
+  - 1!152.*
```

Comment:
I think you should use `1!152` instead of `1!152.*`; not sure whether the `*` works, and that's the equivalent of what e.g. `python` does (saying `3.6` instead of `3.6.5`).

Reply:
thanks @dougalsutherland